patroni/ctl.py

```diff
@@ -628,7 +628,7 @@ def failover(config_file, cluster_name, master, candidate, force, dcs, scheduled
         logging.warning('Failing over to DCS')
         click.echo(timestamp() + ' Could not failover using Patroni api, falling back to DCS')
         click.echo(timestamp() + ' Initializing failover from master {0}'.format(master))
-        dcs.manual_failover(master, candidate, scheduled_at=failover_value)
```

Comment:
You are right. We can't pass string there.
Can you please create a copy of scheduled_at on the line 591 and later use this copy in `dcs.manual_failover` call (line 631)?